tests/integration/test_integration.py

```diff
@@ -133,17 +126,17 @@ async def test_given_no_config_when_post_deployment_tasks_and_authorise_charm_th
         Returns:
             None
         """
-        await self.deploy_charm(ops_test, charm)
         vault_unit = ops_test.model.units["vault-k8s/0"]
         await ops_test.model.wait_for_idle(apps=[APPLICATION_NAME], status="blocked", timeout=1000)
 
-        vault_token = await self.post_deployment_tasks(namespace=ops_test.model_name)
-
+        vault_token = post_deployment_tasks
         await vault_unit.run_action(action_name="authorise-charm", token=vault_token)
+
         await ops_test.model.wait_for_idle(apps=[APPLICATION_NAME], status="active", timeout=1000)
 
+    @pytest.mark.abort_on_fail
     async def test_given_status_is_active_when_run_issue_certificate_action_then_certificates_are_issued(  # noqa: E501
-        self, ops_test, charm, cleanup
+        self, ops_test, build_and_deploy, post_deployment_tasks
```

Comment:
Update docstring to match new args. 
You can also drop the `Returns` section from it.

Reply:
Both done.